core/src/main/java/org/neo4j/ogm/context/MappingContext.java

```diff
@@ -526,4 +526,13 @@ private void generateIdIfNecessary(Object entity, ClassInfo classInfo) {
             primaryIndexField.writeDirect(entity, id);
         }
     }
+
+    public void clear(Object entity) {
```

Comment:
Looks a little bit empty.

Reply:
I used existing methods in the end. Removed this empty method.